Models/account.rb

```diff
@@ -0,0 +1,23 @@
+# user model class
+
+class Account
+  @id
+  @balance
+  @user_id
+  @mysql_obj
+  attr_accessor :balance, :id
+
+  def initialize; end
+
+  def transfer_money(email); end
+
+  def deposit(amount)
+    @mysql_obj.query("UPDATE `accounts` SET `avaliable` = #{@balance + amount} WHERE id = #{@id}")
+    @mysql_obj.query("INSERT INTO `external_trasactions`(`type`, `user_id`, `amount`) VALUES ('deposit',#{@user_id},#{amount})")
+  end
+
+  def withdraw(amount)
+    @mysql_obj.query("UPDATE `accounts` SET `avaliable` = #{@balance - amount} WHERE id = #{@id}")
```

Comment:
Aqui toca agregar un condicional (no se si en SQL o en ruby) para que no deje sacar mas del dinero disponible en la cuenta

Reply:
#1 
```suggestion
    @mysql_obj.query("UPDATE `accounts` SET `avaliable` = #{@balance - amount} WHERE id = #{@id}")
```
@FernandoMorantes  creeria que seria un condicional y ya